packages/javascript/src/index.d.ts

```diff
@@ -1,12 +1,12 @@
 export { protoInt64 } from "@bufbuild/protobuf";
-export * from "./gen/livekit_agent_dispatch_pb.js";
-export * from "./gen/livekit_agent_pb.js";
-export * from "./gen/livekit_egress_pb.js";
-export * from "./gen/livekit_ingress_pb.js";
-export * from "./gen/livekit_metrics_pb.js";
-export * from "./gen/livekit_models_pb.js";
-export * from "./gen/livekit_room_pb.js";
-export * from "./gen/livekit_rtc_pb.js";
-export * from "./gen/livekit_sip_pb.js";
-export * from "./gen/livekit_webhook_pb.js";
-export * from "./gen/version.js";
+export type * from "./gen/dts/livekit_agent_dispatch_pb.d.ts";
```

Comment:
hm, don't remember ever seeing this import pattern 🤔

Reply:
the LSP didn't complain, but i'll double check to make sure this works

Reply:
works great, except the files under `cjs` need to be renamed to `.cjs` in order to be required as CommonJS. i don't know if GH Actions comes built in with any rename utility

Reply:
ok so i did a bulkrename but now i need to sed through all the files and replace the names as well

Reply:
would be nice if we could configure that as part of the protocol es plugin, I think file extension is an option there?

Reply:
only able to set js/ts/d.ts on there unfortunately